bedevere/util.py

```diff
@@ -94,12 +118,50 @@ async def issue_for_PR(gh, pull_request):
     return await gh.getitem(pull_request["issue_url"])
 
 
+def build_pr_body(issue_number: int, body: str) -> str:
+    """Update the Pull Request body with related Issue."""
+    return PR_BODY_TEMPLATE.format(
+        body=body,
+        pr_or_issue_number=issue_number,
+        key=ISSUE.title(),
+        tag_type=ISSUE,
+    )
+
+
+def build_issue_body(pr_number: int, body: str) -> str:
+    """Update the Issue body with related Pull Request."""
+    # If the body already contains a legacy closing tag
+    # (e.g. <!-- /gh-pr-number -->), then we use the legacy template
+    # TODO: Remove this when all the open issues using the legacy tag are closed
+    if PR_BODY_CLOSING_TAG.format(tag_type=PR) in body:
```

Comment:
I am leaving the issues updated with the old implementation (e.g. `<!-- /gh-pr-number -->`) alone.